src/apps/front-office/design-system/layouts/Header/MiddleHeader.tsx

```diff
@@ -0,0 +1,120 @@
+import { Link } from "@mongez/react-router";
```

Comment:
Please make sure to generate the component using `Generator Z` or keep the component structure same as the rest of the project